framework/src/postprocessors/ChangeOverTimestepPostprocessor.C

```diff
@@ -0,0 +1,46 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#include "ChangeOverTimestepPostprocessor.h"
+
+template<>
+InputParameters validParams<ChangeOverTimestepPostprocessor>()
+{
+  InputParameters params = validParams<GeneralPostprocessor>();
+  params.addRequiredParam<PostprocessorName>("postprocessor", "The name of the postprocessor");
+  return params;
+}
+
+ChangeOverTimestepPostprocessor::ChangeOverTimestepPostprocessor(const InputParameters & parameters) :
+    GeneralPostprocessor(parameters),
+    _pps_value(getPostprocessorValue("postprocessor")),
+    _pps_value_old(getPostprocessorValueOld("postprocessor"))
+{
+}
+
+void
+ChangeOverTimestepPostprocessor::initialize()
+{
+}
+
+void
+ChangeOverTimestepPostprocessor::execute()
+{
+}
+
+Real
+ChangeOverTimestepPostprocessor::getValue()
+{
+  return _pps_value - _pps_value_old;
```

Comment:
We typically compute the PPS value in `execute()` and `getValue` acts like a getter. Typically it has some parallel communication in it as well. I guess this will work in parallel, since `_pps_value` and `_pps_value_old` are the same on all procs.

In case parallel test fails, this might be a source of problems.